lib/charms/prometheus_k8s/v0/prometheus.py

```diff
@@ -317,7 +317,7 @@ def _on_scrape_targets_changed(self, event):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 4
```

Comment:
I would not mind if you leave it at 4. Since we have not released to edge and are still accruing features. Your call.

Reply:
On second thoughts I would like to request you to leave it at 4 since otherwise we will have missing version numbers in our release history for this charm library.